pkgs/servers/routinator/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, rustPlatform, rsync }:
```

Comment:
@0x4A6F I just noticed you have `rsync` here but it isn't used anywhere.

Reply:
@aanderse Removed `rsync`.